diff --git a/src/Web/Kernel.php b/src/Web/Kernel.php
index 976f61821778dc0197c7d1f0506f2cbadc094ad8..1dc0d75f057e5fea95540105e3f880721318d7b9 100644
--- a/src/Web/Kernel.php
+++ b/src/Web/Kernel.php
@@ -1,5 +1,8 @@
 <?php
 namespace Bamboo\Web;
 class Kernel {
-  public array $middleware = [\Bamboo\Web\Middleware\SignatureHeader::class];
+  public array $middleware = [
+    \Bamboo\Web\Middleware\RequestId::class,
+    \Bamboo\Web\Middleware\SignatureHeader::class,
+  ];
 }
